Anthony/buy_me_a_laptop_challenge.py

```diff
@@ -0,0 +1,104 @@
+# Challenge 1
+print("Challenge 1: All possible laptops\n")
+
+print("Question 1: You are given a list containing the laptop names. Print the names of each the laptops separately.")
+all_laptops = ["Apple MacBook Pro", "Asus Zenbook", "Dell XPS", "Lenovo IdeaPad", "Apple MacBook Air", "Sony Viao"]
+
+
+# TODO: Write code to print all laptop names
+print("All laptops names:")
+
+
```

Comment:
We can revisit this, but in this case you could have added print statements for each item, **or use an f -string to combine all laptops into one sentence???**